assets/js/theme/search.js

```diff
@@ -79,6 +86,57 @@ export default class Search extends CatalogPage {
         urlUtils.goToUrl(url);
     }
 
+    activateTab($tabToActivate) {
+        const $tabsCollection = $('[data-search-page-tabs]').find('[role="tab"]');
+
+        $tabsCollection.each((idx, tab) => {
+            const $tab = $(tab);
+
+            if ($tab.is($tabToActivate)) {
+                $tab.removeAttr('tabindex');
+                $tab.attr('aria-selected', true);
+                const activeTabPanelSelector = `#${$tab.attr('aria-controls')}`;
+                $(activeTabPanelSelector).attr('tabindex', 0);
```

Comment:
In my opinion it is better to write self-declared code without comments in simple cases. It is better to read statement with understandable variable name instead of long expression. But if you insist I can change it - not so critical :)

Reply:
Self declared code means not creating unnecessary variables which are using as code comments

Reply:
removed